src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -939,6 +939,17 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
                     } else {
                         "DoNotSchedule".to_string()
                     },
+                    // Consider node affinities when calculating topology spread. This is the
+                    // default: <https://docs.rs/k8s-openapi/latest/k8s_openapi/api/core/v1/struct.TopologySpreadConstraint.html#structfield.node_affinity_policy>,
+                    // made explicit.
+                    node_affinity_policy: Some("Honor".to_string()),
+                    // Do not consider node taints when calculating topology spread. This is the
+                    // default: <https://docs.rs/k8s-openapi/latest/k8s_openapi/api/core/v1/struct.TopologySpreadConstraint.html#structfield.node_affinity_policy>,
+                    // made explicit.
+                    node_taints_policy: Some("Ignore".to_string()),
```

Comment:
Wow, that's crazy that Ignore is the default. I'd assume that we'd want it to honor taints, but maybe there is stuff I'm not thinking about here.

Not going to block the merge for this, especially since I don't fully understand the implications of honoring it.

Reply:
yeah, i don't understand the implications either